bitnami/rabbitmq/Chart.yaml

```
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/rabbitmq
   - https://www.rabbitmq.com
-version: 11.4.0
+version: 11.4.1
```

As it is something new, I think it should be a minor.
```suggestion
version: 11.5.0
```

---

Seems reasonable to me.

Do I have to make a new commit regarding this change?
Sorry, this is my first time contributing.

---

You can click the button "Sign off and commit suggestion" to resolve the comment, after that I can merge the PR